packages/workspace/src/utils/cli-config-utils.ts

```diff
@@ -34,6 +34,10 @@ export function serializeTarget({ project, target, config }) {
   return [project, target, config].filter((part) => !!part).join(':');
 }
 
+/**
+ *
+ * @deprecated This function will be removed from `@nrwl/workspace` in version 17. Prefer importing from `@nrwl/devkit`
+ */
 export function replaceAppNameWithPath(
```

Comment:
I don't think anyone is using this:

https://grep.app/search?q=replaceAppNameWithPath&case=true&filter[lang][0]=TypeScript

Let's remove it.